test/package-test.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+const Hapi = require('hapi');
+const lab = exports.lab = require('lab').script();
+const code = exports.code = require('code');
+const hapiSlack = require('../');
+let server;
+
+lab.afterEach((done) => {
+  server.stop(() => {
+    done();
+  });
+});
+lab.test('converts a basic message passed as string ', (done) => {
+  server = new Hapi.Server({});
+  server.connection({ port: 8080 });
+  server.register({
+    register: hapiSlack,
+    options: {}
+  }, () => {
+    const packetString = server.makeSlackPayload(['test'], 'a string');
+    code.expect(typeof packetString).to.equal('string');
+    const packet = JSON.parse(packetString);
+    code.expect(packet.attachments.length).to.equal(1);
+    code.expect(packet.attachments[0].fields.length).to.equal(1);
+    code.expect(packet.attachments[0].title).to.equal('a string');
+    code.expect(packet.attachments[0].fields[0].title).to.equal('Tags');
+    done();
+  });
+});
+lab.test('lets you post an object as the message', (done) => {
+  server = new Hapi.Server({});
```

Comment:
you could move this into setup. so each test is just calling server.makeSlackPayload